tests/test_parsl_flux_mpi_hello.py

```diff
@@ -85,20 +85,21 @@ def run_mpi_pi(
 
 # Set up fixture to initialize and cleanup Parsl
 @pytest.fixture(scope="module")
-def load_config(config_file, request):
+def conf(config_file, platform):
```

Comment:
I would change the name of this method to `config`. I don't think it needs to be shortened

Reply:
I don't really like `conf` either.